syne_tune/optimizer/schedulers/multiobjective/expected_hyper_volume_improvement.py

```diff
@@ -0,0 +1,336 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from typing import Optional, List, Dict, Any, Union
+import logging
+
+import numpy as np
+
+from syne_tune.try_import import try_import_botorch_message
+
+try:
```

Comment:
Not for now and this PR but we really need to get rid of this try import pattern.

Reply:
+1